### ISSUE TITLE
Installation Error

### ISSUE THREAD

**Issue by RicoHeat** [Divested-Mobile/Mull-Fenix#11]
Attempted downloading and installing apk from F-Droid but I always get an error.. Downloaded F-Droid app and tried again but for some reason I get installation errors with no explanation from system.. 
Android 9 armv7

Is it possible that since I have Iceraven Browser already installed that there is some kind of conflict?

**Comment by SkewedZeppelin** [comment]
Do you already have Mull installed?

Mull from DivestOS repo and Mull from F-Droid official repo use
different keys. You have to upgrade from the corresponding repo if so.

**Comment by RicoHeat** [comment]
> Do you already have Mull installed? Mull from DivestOS repo and Mull from F-Droid official repo use different keys. You have to upgrade from the corresponding repo if so.

No @SkewedZeppelin.. I was just doing a clean install.. I have never used the app before.. I was just wanting to check it out.

**Comment by SkewedZeppelin** [comment]
Strange.
There are two of each version, one for arm and one for aarch64.
In the F-Droid app listing, the aarch64 one should always be higher.
Please try manually selecting the second one of the latest version, and try if that changes.

**Comment by SkewedZeppelin** [comment]
You closed this issue?

Were you eventually able to install the app?

**Comment by RicoHeat** [comment]
@SkewedZeppelin 
Yes I now have it installed.. I was initially downloading and trying to install the 64 version without even knowing it.. 
I just assumed that the version at the top of the list was the latest.. It would be nice if each version had a tag or label that indicates which one is Armv7 or 64.. whatever the case may be so that people like me don't make the same mistakes 🙃
Thanks for your help 👍
